Test Try action
before
try
catch: try
finally
catch from test: catch
